src/app/pages/dashboard/components/widget-sys-info/widget-sys-info.component.html

```diff
@@ -247,11 +288,21 @@ <h3>
                   <mat-list-item *ngIf="uptimeString">
                     <strong>{{ 'Uptime' | translate }}:</strong>
                     <span>
-                      {{ '{uptimeString} as of {dateTime}' | translate: { uptimeString, dateTime } }}
+                      {{
+                        '{uptimeString} as of {dateTime}'
```

Comment:
I kinda like pipes on the same line. Same goes for ternary conditions.

Reply:
sorry, had prettier ON